change/@azure-msal-common-2020-08-20-15-48-02-claims-request-tokenendpt-v2.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Add claims request to /token calls (#2138)",
+  "packageName": "@azure/msal-common",
+  "email": "thomas.l.norling@gmail.com",
```

Comment:
We decided these should be our Microsoft emails, right?

Reply:
Thanks. I dont know why it's still doing this. I changed it and I guess it changed itself back